VideoLocker/res/values/styles.xml

```diff
@@ -42,6 +42,7 @@
         <item name="android:progressBarStyleInverse">@style/edX.Widget.ProgressBar.Inverse</item>
         <item name="android:progressBarStyleSmallInverse">@style/edX.Widget.ProgressBar.Small.Inverse</item>
         <item name="android:progressBarStyleLargeInverse">@style/edX.Widget.ProgressBar.Large.Inverse</item>
+        <item name="alertDialogTheme">@style/AppTheme.Dialog.Alert</item>
```

Comment:
Looks like we we weren't using the code that was reference the old AlertDialog, exept in one place, `CustomSelectView`, which is being removed in https://github.com/edx/edx-app-android/pull/815